projects/postgis/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./autogen.sh
+./configure --enable-static --without-raster --without-protobuf
+cd liblwgeom
+make clean -s
+make -j$(nproc) -s
+cd ..
+
+./fuzzers/build_google_oss_fuzzers.sh
```

Comment:
`make fuzzers`, `make fuzz_targets` , or some such would be great. 
What we are tying to encourage is that the fuzz targets become part of the project's continuous integration process, even w/o actually fuzzing. 
Read more at https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md